mod_app/static/css/film.css

```diff
@@ -0,0 +1,115 @@
+/* for film detail and list */
+/* list */
+.film-table {
+  font-size: 0.9rem;
+  .synopsis > * {
+    min-width: 25vw;
+    font-size: inherit;
+  }
+  .synopsis .open-icon {
+    font-size: inherit;
+    display: inline;
+    align-content: center;
+  }
+  .cast > * {
+    font-size: inherit;
+  }
+}
+.film-table th {
+  font-size: 1rem;
+}
+
+/* detail */
+.top-level {
+  display: flex;
+}
+.video {
+  display: inline;
+  > video {
+    height: 40vh;
+    width: 40vw;
+  }
+}
+.triple-c {
+  display: inline-flex;
+  /* justify-content: space-between; */
+  flex-direction: column;
+  align-content: center;
+  margin-left: 0.4rem;
+  > dt {
+    margin: 0.4rem 0;
+  }
+}
+
+.genres {
```

Comment:
Did you mean genre? I can't find genres on this page

Reply:
Ignore this, I thought I was looking at the admin page.

Reply:
it's part of the detail page - it's for a <dl> element